main.go

```diff
@@ -216,10 +214,19 @@ func init() {
 		GroupID: groupDev,
 		RunE:    withValidProjectURI(cmd.SessionCreateCmd),
 	}
+
 	newCmd.Flags().StringVarP(&config.BuildID, "image", "i", "", "Build ID of the container image to use")
 	newCmd.Flags().StringVar(&config.Provider, "provider", "", "Provider to use")
-	newCmd.Flags().StringVar(&config.NodeTypeID, "type", "", "Node type to use, eg. `gpu_1x_a100`")
+
```

Comment:
Delete empty line. These flags should also be on the any other command that creates new sessions (`code`, `ssh`)